### PR TITLE
Fix interpunction, mention scripts in -A.

### PR DESCRIPTION
**Pull request by d33tah**

### CODE REVIEW

**Comment by igorshubovych** [comment]
We don't put colons at end of command description.

**Comment by d33tah** [comment]
@igorshubovych: this wasn't by your CI then. Changed to dots - wanted that to be uniform.

File: pages/common/nmap.md [hunk 1]
```diff
@@ -11,18 +11,18 @@
...
 
 `nmap -A {{address_or_addresses}}`
 
-- Assume good network connection and speed up execution:
+- Assume good network connection and speed up execution.
 
 `nmap -T4 {{address_or_addresses}}`
 
-- Scan a specific list of ports (use -p- for all ports 1-65535)
+- Scan a specific list of ports (use -p- for all ports 1-65535).
```
Comment (igorshubovych) [inline pages/common/nmap.md]: Don't use this trick to have +1 code example in parens.
If the example is common and useful add it separately.

File: pages/common/nmap.md [hunk 2]
```diff
@@ -11,18 +11,18 @@
...
 
 `nmap -p {{port1,port2,...,portN}} {{address_or_addresses}}`
 
-- Perform TCP and UDP scanning (use -sU for UDP only, -sZ for SCTP, -sO for IP):
+- Perform TCP and UDP scanning (use -sU for UDP only, -sZ for SCTP, -sO for IP).
```
Comment (igorshubovych) [inline pages/common/nmap.md]: Don't use this trick to have +1 code example in parens.
If the example is common and useful add it separately.

Comment (igorshubovych) [inline pages/common/nmap.md]: It is better to break 5-6 examples rules, then write something like this.

**Comment by rubenvereecken** [comment]
While @igorshubovych's comments are justified, the changes in question are actually [two days old](https://github.com/tldr-pages/tldr/commits/master/pages/common/nmap.md). I suggest we merge this in as it is and patch in fixes for those comments in a separate PR.

**Comment by igorshubovych** [comment]
@rubenvereecken go for it

**Pull request merged**